### ISSUE TITLE
[steamshovel] I3MCTree with zero emission points (Trac #1426)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1418]
make option to remove the emission points along the tracks from a I3MCTree,
current option to just set 'scale' to 1 is kind of ugly.
possibly allow for minimum value 0 and catch this case in the code.

Migrated from https://code.icecube.wisc.edu/ticket/1426
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:08", 
    "description": "make option to remove the emission points along the tracks from a I3MCTree,\ncurrent option to just set 'scale' to 1 is kind of ugly.\npossibly allow for minimum value 0 and catch this case in the code.", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335648518728", 
    "component": "combo core", 
    "summary": "[steamshovel] I3MCTree with zero emission points", 
    "priority": "normal", 
    "keywords": "steamshovel I3MCTree", 
    "time": "2015-11-06T08:49:01", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151106T14:54:10: hdembinski changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151106T15:17:21: 
* hdembinski commented:

> In changeset 139263:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139263"
> allow scale and power to be zero, closes #1418
> ```


* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:08: 
* olivas commented:

> Milestone December 2015 deleted


* olivas changed milestone from "December 2015" to ""
